src/themes/default/components/core/blocks/Auth/Login.vue

```diff
@@ -79,10 +79,10 @@ import ButtonFull from 'theme/components/theme/ButtonFull.vue'
 import BaseCheckbox from '../Form/BaseCheckbox.vue'
 import BaseInput from '../Form/BaseInput.vue'
 import { required, email } from 'vuelidate/lib/validators'
-import i18n from '@vue-storefront/i18n'
+import { Notification } from '@vue-storefront/core/modules/notification/components/Notification'
```

Comment:
Instead of mixing in whole Notification component with 2/3 unused parts we can just dispatch appropiate action.
` this.$store.dispatch('notification/addNotification', notification)`